nativelib/src/main/resources/posix/sys/socket.c

```diff
@@ -104,21 +104,20 @@ int scalanative_connect(int socket, struct scalanative_sockaddr *address,
     return result;
 }
 
-int scalanative_bind(int socket, struct scalanative_sockaddr *address,
-                     socklen_t address_len) {
+int scalanative_connect(int socket, struct scalanative_sockaddr *address,
+                        socklen_t address_len) {
     struct sockaddr *converted_address;
     int convert_result =
         scalanative_convert_sockaddr(address, &converted_address, &address_len);
 
     int result;
 
     if (convert_result == 0) {
-        result = bind(socket, converted_address, address_len);
+        result = connect(socket, converted_address, address_len);
```

Comment:
The diff is confusing in this file. Have you just switched the order in which `scalanative_connect` and `scalanative_bind` are defined?